src/main/kotlin/stringcalculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,27 @@
+package stringcalculator
+
+import java.lang.IllegalArgumentException
+
+class StringAddCalculator {
+
+    fun add(text: String): Int {
+        if (text.isBlank()) return 0
+        if (text.isEmpty()) return 0
+        if (text.length == 1) return text.toInt()
+        val splittext = text.split(",", ":")
+        val result = Regex("//(.)\n(.*)").find(text)
+        result?.let {
+            val customDelimiter = it.groupValues[1]
+            val tokens = it.groupValues[2].split(customDelimiter)
+            tokens.forEach{if (it.toInt() < 0) throw RuntimeException("$it is negative") }
+            return tokens.map { it.toInt() }.sum()
+        }
+        splittext.forEach{if (it.toInt() < 0) throw RuntimeException("$it is negative") }
+        return splittext.map { it.toInt() }.sum()
+    }
+    fun negativecheck(number: Int){
```

Comment:
Kotlin도 Java와 동일하게 camelCase를 사용합니다.

```suggestion
    fun negativeCheck(number: Int){
```

Reply:
추가로 전체적으로 ktlint가 하나도 적용되지 않은 것으로 보입니다.

아래 메뉴에서 ktlint 설정을 확인하신 뒤 추가 후 활용해주시면 어떨까요?

![image](https://user-images.githubusercontent.com/34656258/89671574-2e76b800-d91e-11ea-9cf9-c84f63360635.png)